src/RatkoR/Crate/Connection.php

```diff
@@ -195,4 +197,43 @@ public function query()
             $this, $this->getQueryGrammar(), $this->getPostProcessor()
         );
     }
+
+   /**
+     * Crate works with extended fields like arrays and objects. If
+     * exception is triggered, laravel logs SQL and
+     * it's parameters. While doing this it casts parameters to
+     * string which then fails in it's default QueryException with:
+     *    ErrorException: Object of class stdClass could not be converted to string
+     *
+     * We're overriding runQueryCallback to trigger our own
+     * version of QueryException.
+     * 
+     * @param  string    $query
+     * @param  array     $bindings
+     * @param  \Closure  $callback
+     * @return mixed
+     *
+     * @throws \Illuminate\Database\QueryException
+     */
+    protected function runQueryCallback($query, $bindings, Closure $callback)
+    {
+        // To execute the statement, we'll simply call the callback, which will actually
+        // run the SQL against the PDO connection. Then we can calculate the time it
+        // took to execute and log the query SQL, bindings and time in our memory.
+        try {
+            $result = $callback($this, $query, $bindings);
+        }
+
+        // If an exception occurs when attempting to run a query, we'll format the error
+        // message to include the bindings with SQL, which will make this exception a
+        // lot more helpful to the developer instead of just the database's errors.
+        catch (Exception $e) {
```

Comment:
Fixed. Seems it was too late last night... Thx!

The test is also fixed and now properly catches RatkoR\Crate\QueryException. Any other issue you might see?